### ISSUE TITLE
Folders with ".swift" in name break extractLocStrings

### ISSUE THREAD

**Issue by tomislav** [Flinesoft/BartyCrouch#159]
I use a Pod called "MapboxGeocoder.swift". When running BartyCrouch this seems to break the "update code" task since extractLocStrings thinks the folder is a file.

```
➜ bartycrouch update
Starting Task 'Update Interfaces' ...
2019-12-30 14:16:44.658: ✅  ./ABC/Base.lproj/SubscriptionViewController.xib:  Successfully updated strings file(s) of Storyboard or XIB file.
Task 'Update Interfaces' took 4.452 seconds.
Starting Task 'Update Code' ...
extractLocStrings: error: failed to read input file /Users/tomislav/Documents/Projects/ABC/ABC-ios/Pods/Target Support Files/MapboxGeocoder.swift
2019-12-30 14:16:46.193: ❌  Could not extract strings from Code in directory '.'.
Task 'Update Code' took 1.438 seconds.
Starting Task 'Normalize' ...
Task 'Normalize' took 0.242 seconds.
```

**Comment by Jeehut** [comment]
@tomislav Could you please provide your full config file? It looks like you're using the default `.` set for all the different `path` options. It is strongly recommended that you specify more specific folders for the `path` options which might also fix your issue.

But if it's in the `Pod` directory, I would say this is a duplicate of #152.

**Comment by tomislav** [comment]
Yes, I'm using "." and it's in the Pods folder.

The problem is that my app is structured like this:

```
Root
-- iOS app
-- Watch app
-- Watch extension
-- Watch intent extension
-- Intent extension
-- Pods
```

So it hard to be more specific without support for multiple codePaths. I see #155 and #156 mention this.

I also like my Pods localized, since they sometimes contain strings that will be displayed to the user 🤷‍♂

**Comment by tomislav** [comment]
Fixed with #167

**Comment by Jeehut** [comment]
You're right, forgot to close this one. Thanks to @fredpi! 💪
